src/lein_tools_deps/deps.clj

```diff
@@ -101,10 +76,10 @@
 
 (defn make-deps
   "Reads and merges all of the deps-ref, returning a single deps map"
-  [{:keys [root] {:keys [config-files] :as config} :lein-tools-deps/config :as project}]
+  [exists? read-deps env {:keys [root] {:keys [config-files]} :lein-tools-deps/config}]
```

Comment:
I think if we make these default to `env/exists?` and `reader/read-deps` and have them applied in an arity 2 function call, we should merge this.